$NetBSD: patch-ak,v 1.1 2005/10/27 00:18:51 rillig Exp $

What happens if you call two parameters by the same name? It's a
compile-time error. What a surprise.

--- modules/speller/default/affix.hpp.orig	2004-11-29 18:50:06.000000000 +0100
+++ modules/speller/default/affix.hpp	2005-10-27 02:11:29.724080400 +0200
@@ -107,9 +107,9 @@ namespace aspeller {
     {
       return expand(word,aff,buf,0);
     }
-    WordAff * expand_suffix(ParmString word, const unsigned char * new_aff,
+    WordAff * expand_suffix(ParmString word, const unsigned char *,
                             ObjStack &, int limit = INT_MAX,
-                            unsigned char * new_aff = 0, WordAff * * * l = 0,
+                            unsigned char * = 0, WordAff * * * l = 0,
                             ParmString orig_word = 0) const;
     
   private:
